lib/prometheus_exporter/client.rb

```diff
@@ -119,8 +119,9 @@ def process_queue
     end
   end
 
-  def stop
+  def stop(wait_timeout_seconds = 10)
```

Comment:
It would be nicer here not to change the default behavior and instead of that us passing in the time where we actually need it.